src/server/schemas/search_taxa.yaml

```diff
@@ -2,10 +2,6 @@ type: object
 required: [search_text, ns]
 additionalProperties: false
 properties:
-  no_count:
-    type: boolean
-    default: false
-    description: Skip counting the total search results
```

Comment:
No one is using `search_taxa` correctly. There is actually a probably better way to get the count too, if we really need it.